js/webpack/libraries.js

```diff
@@ -30,7 +30,7 @@ module.exports = {
     // library
     'inject': ['./inject.js'],
     'web3': ['./web3.js'],
-    'parity': ['./parity.js']
+    'parity': ['./library.parity.js']
```

Comment:
We have things mixed up here. This version of parity is supposed to be the version served via `/parity-utils/{inject,parity}.js` - library refers to the npm version.